### DISCUSSION TITLE
Installing Podman via brew doesn't work

### DISCUSSION THREAD

**Discussion by StarpTech** [containers/podman#15334] [General]
It installs everything but `podman` is not reachable after that. Other formulas works fine.
```sh
brew install podman
```

OS: Ubuntu
ARCH: x86_64

**Comment by rhatdan** [comment]
You are doing `brew install podman` on an Ubuntu Machine?

brew is intended for MACOS

**Reply by afbjorklund** [reply]
It should install `podman-remote` also on Linux, but that is usually **not** what you want (although `podman machine` _should_ work)

```console
$ brew install podman
$ podman-remote machine init --now
```

There were some efforts to package the "real" podman (for Linux) in brew, but no takers: https://github.com/Homebrew/homebrew-core/pull/84276

Note: the `podman-remote` binary is renamed to just `podman`, when running on macOS

**Reply by afbjorklund** [reply]
I tested the `podman` formula on Ubuntu 22.04, and besides downloading a couple of GiB for `qemu` it seemed to work...

```console
$ brew install podman
...
==> Installing dependencies for podman: llvm, gzip, libpciaccess, libdrm, wayland, libva, libvdpau, libxshmfence, libxv, libxvmc, lm-sensors, wayland-protocols, mesa, mesa-glu, libxinerama, xinput, freeglut, libepoxy, freetype, fontconfig, cairo, fribidi, graphite2, harfbuzz, pango, dbus, libxtst, at-spi2-core, at-spi2-atk, iso-codes, xkeyboardconfig, libxkbcommon, gtk+3, libcap-ng and qemu
...
$ podman-remote --version
podman-remote version 4.2.0
$ podman-remote machine init
Downloading VM image: fedora-coreos-36.20220806.2.0-qemu.x86_64.qcow2.xz: done  
Extracting compressed file
Image resized.
Machine init complete
To start your machine run:

	podman machine start

$ podman-remote machine start
Starting machine "podman-machine-default"
Waiting for VM ...
...
Machine "podman-machine-default" started successfully
$ podman-remote version
Client:       Podman Engine
Version:      4.2.0
API Version:  4.2.0
Go Version:   go1.18.5
Built:        Wed Aug 10 22:46:05 2022
OS/Arch:      linux/amd64

Server:       Podman Engine
Version:      4.1.1
API Version:  4.1.1
Go Version:   go1.18.4
Built:        Fri Jul 22 21:05:59 2022
OS/Arch:      linux/amd64
```

It will use a VM for running Podman Engine, but this is the same as on Mac and when using Docker Desktop on Linux ?

```console
$ sudo apt install podman
...
The following additional packages will be installed:
  buildah catatonit conmon containernetworking-plugins crun fuse-overlayfs golang-github-containernetworking-plugin-dnsname golang-github-containers-common
  golang-github-containers-image libostree-1-1 uidmap
...
$ podman version
WARN[0000] Failed to decode the keys ["configmaps"] from "/home/anders/.config/containers/containers.conf". 
WARN[0000] Failed to decode the keys ["configmaps"] from "/home/anders/.config/containers/containers.conf". 
WARN[0000] Failed to decode the keys ["configmaps"] from "/home/anders/.config/containers/containers.conf". 
WARN[0000] Failed to decode the keys ["configmaps"] from "/home/anders/.config/containers/containers.conf". 
Version:      3.4.4
API Version:  3.4.4
Go Version:   go1.17.3
Built:        Thu Jan  1 01:00:00 1970
OS/Arch:      linux/amd64

```

**Comment by StarpTech** [comment]
No, I use https://docs.brew.sh/Homebrew-on-Linux. Please reopen the issue.

**Reply by rhatdan** [reply]
This is a discussion.  No one builds the image on Homebrew-on-Linux upstream that I know of, so this is not an upstream issue.

**Reply by StarpTech** [reply]
Brew is not important to me. I'm looking for a way to install the latest version 4.2 on Linux.

**Reply by rhatdan** [reply]
Can you install the debian packages?

**Reply by StarpTech** [reply]
Yes, but they are shipping an old version of podman. https://github.com/containers/podman/discussions/15333

**Reply by rhatdan** [reply]
Then you either need to wait, or open an issue with them or update to a newer version of the OS or build from source.
There is no one currently offering a repo for Ubuntu users that is kept up to date, that I know of.
@lsm5 WDYT?

**Reply by StarpTech** [reply]
I opened an issue https://github.com/containers/podman/issues/15330

> Then you either need to wait

According to the state, nobody is maintaining it.

> update to a newer version of the OS

I'm on the latest version 22.10.

> or build from source

This is not easy according to the instruction and it will not setup podman for me properly on the machine.

**Reply by lsm5** [reply]
@StarpTech https://github.com/containers/podman/discussions/15333#discussioncomment-3422339
